### ISSUE TITLE
would love to have an option to backup using ssh instead of https

### ISSUE THREAD

**Issue by wanghanlin** [itsTeknas/GitlabBackupUtil#9]

**Comment by nstohler** [comment]
I've changed the last part of the script (since it didn't work over https) to the following to get it over SSH:

```
  for (let repo of pgits) {
    const repoName = repo.substring(19, repo.length - 4)
    const sshRepoName = `git@gitlab.com:${repoName}`
    console.log(`Cloning ${repoName} | ${sshRepoName}`)

    const stdout = await cmdAsync(`git clone ${sshRepoName} ${argv.output || 'gitlab-backup'}/${repoName}`).catch(console.log)
    // console.log(stdout)
    index++
    cloneProgressBar.update(index)
  }
```

NOTE:
I had to edit this directly inside 
`C:\Users\[WinUserName]\AppData\Roaming\npm\node_modules\gitlab-backup-util\index.js`
for it to work.

**Comment by pyropy** [comment]
Hi, merge request #14 resolves this.

**Comment by itsTeknas** [comment]
Merged

**Comment by itsTeknas** [comment]
published in 1.0.4
